config/crd/bases/core.openfeature.dev_featureflagsources.yaml

```diff
@@ -0,0 +1,303 @@
+---
```

Comment:
Is it convention to pluralize the name and filename? `featureflagsources` instead of `featureflagsource`?

Reply:
this is a completely autogenerated file, it is expected here